api/api_test.go

```diff
@@ -29,3 +37,94 @@ func testHTTPServer(
 
 	return config, ln
 }
+
+// nextPort is the next port to use for the API server.
+var nextPort int32 = 28200
+
+// restVaultServer runs an instance of the Vault server in development mode.
+// This requires that the vault binary is installed and in the $PATH.
+func testVaultServer(t *testing.T) (*Client, func()) {
```

Comment:
To avoid a cyclic dependency in the tests you could make the test file's package name `package api_test`.